### DISCUSSION TITLE
Potential breakout: Future theoretical trends

### DISCUSSION THREAD

**Discussion by ml-evs** [MADICES/MADICES-2022#22] [Ideas]
Just jotting this down briefly while I have time, can expand later (hopefully).

Since we have assembled some real experts on data modelling, I think a room where (primarily) they discuss any new theoretical developments they are aware of/involved in that will maybe help towards our goals in the next 10-20 years, e.g., extensions or simplications of RDF, more prolific use of graph data modelling (not exactly new!), category theory for unification of relational/document-based/graph data (https://www.categoricaldata.net/), or even new/alternative philosophical frameworks for ontology development.

**Comment by kjappelbaum** [comment]
A tangentially related theme: How do we educate people at this interface (i.e. to be aware of such theoretical developments but also have some idea about chemistry/materials science). How can we facilitate exchange/remove "language" barriers.
